src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,31 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTab, setSelectedTab }) => (
+  <div data-cy="TabsComponent">
+    <div className="tabs is-boxed">
+      <ul>
+        {tabs.map(tab => (
+          <li
+            className={cn({
+              'is-active': selectedTab.id === tab.id,
+            })}
+            data-cy="Tab"
+            key={tab.id}
+          >
+            <a
+              href={`#${tab.id}`}
+              data-cy="TabLink"
+              onClick={() => setSelectedTab(tab)}
```

Comment:
```suggestion
              onClick={setSelectedTab(tab)}
```
see comment in App component